cloudify/workflows/workflow_context.py

```diff
@@ -443,6 +443,12 @@ def instances(self):
         """The node instances"""
         return iter(self._node_instances.values())
 
+    def has_operation(self, operation_interface):
+        try:
+            return bool(self.operations[operation_interface]['operation'])
```

Comment:
yes, that means it's not there, for the purposes of this check. Currently nodes have all the operations declared-but-empty. (it's not actually an empty dict, it's rather, `{"plugin": "", "operation": "", ...}`)

But maybe someday we'll replace it with the dicts not even being there, rather than being empty-ish. Declaring all of the nonexistent operations is a big part of the blueprint plan when dumped to json.